packages/jaeger-ui/src/constants/default-config.tsx

```diff
@@ -20,6 +20,10 @@ export default deepFreeze(
   Object.defineProperty(
     {
       archiveEnabled: false,
+      // TODO: remove default
+      deepDependencies: {
+        menuEnabled: true,
```

Comment:
Do we really want that in oss?  There's no backend to generate DDGs.

Reply:
yeah hence the TODO and comment above that I thought TODOs were dealt with...
definitely won't have this on by default. same with the `New!` css

Reply:
Fixed